src/reconcile/rec_write.c

```diff
@@ -2644,8 +2651,10 @@ __rec_write_image(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WTI_REC_CHUNK *chu
                 multi->block_meta->backlink_lsn = multi->block_meta->disagg_lsn;
             multi->block_meta->delta_count = 0;
             multi->block_meta->base_lsn = WT_DISAGG_LSN_NONE;
-        } else
+        } else {
             __wt_page_block_meta_assign(session, multi->block_meta);
```

Comment:
There is another call to `__wt_page_block_meta_assign`, should it set `WT_REC_DISAGG_NEW_PAGE` too?

Reply:
good catch. Will fix.

Reply:
fixed